### PR TITLE
[release-4.8][submodule] Update 

### PR DESCRIPTION
**Pull request by sebsoto**
* [submodule][kube-proxy] Update to https://github.com/openshift/kubernetes/commit/43b818f2e5a3be1f85e796c5f7430c10427f826c
* [submodule][kubelet] Update to https://github.com/openshift/kubernetes/commit/98073871f173baaa04dc2bafab50effd62c308a6
* [submodule][containernetworking-plugins] Update to https://github.com/openshift/containernetworking-plugins/commit/71a8b3469df865daf68cb893bfe313328d4be992
* [submodule][ovn-kubernetes] Update to https://github.com/openshift/ovn-kubernetes/commit/7da97fcfffb1770d980a85b5667b78d5c7f67c95

Also includes cleaning up go.mod by removing necessary replace directives.

### CODE REVIEW

**Review by jrvaldes — APPROVED** [review]
/lgtm

File: go.mod [hunk 1]
```diff
@@ -10,7 +10,7 @@ replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20201214114959-164a2fb63b5f // OpenShift 4.7
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20201214125552-e615e336eb49 // OpenShift 4.7
```
Comment (aravindhp) [inline go.mod:11]: Do we need to replace this with 4.8?

Comment (sebsoto) [reply go.mod:11]: I am unable to update the api library:
```
[sebsoto@sdesktop windows-machine-config-operator]$ go get -u github.com/openshift/api@be21c6cb1bea11427ff13ef95e155aa04b7c5939
github.com/openshift/api imports
	k8s.io/api/batch/v2alpha1: cannot find module providing package k8s.io/api/batch/v2alpha1
```

Comment (aravindhp) [reply go.mod:11]: Did you trying deleting the two lines and running `go mod tidy && go mod vendor && go mod verify`?

```patch
diff --git a/go.mod b/go.mod
index 2767e881..12156208 100644
--- a/go.mod
+++ b/go.mod
@@ -7,8 +7,6 @@ replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503
 replace (
        github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.38.1-0.20200424145508-7e176fda06cc
        github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.10.0
-       github.com/openshift/api => github.com/openshift/api v0.0.0-20201214114959-164a2fb63b5f // OpenShift 4.7
-       github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20201214125552-e615e336eb49 // OpenShift 4.7
 
        k8s.io/api => k8s.io/api v0.22.0-rc.0
        k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.22.0-rc.0
```

Comment (sebsoto) [reply go.mod:11]: Doing that results in picking up commits from may, when we want the latest release-4.8 commits
```
[sebsoto@sdesktop windows-machine-config-operator]$ go mod tidy && go mod vendor && go mod verify
go: downloading github.com/openshift/api v0.0.0-20210521075222-e273a339932a
go: downloading github.com/openshift/client-go v0.0.0-20210521082421-73d9475a9142
all modules verified
```

Comment (aravindhp) [reply go.mod:11]: I am fine with that. Better than release-4.7. We can revisit bumping to the latest in a separate PR.

**Review by aravindhp — APPROVED** [review]
Nice work, @sebsoto. This really cleans up the go.mod.

/approve

File: go.mod [hunk 2]
```diff
@@ -2,37 +2,10 @@ module github.com/openshift/windows-machine-config-operator
...
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.21.1
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.21.1
-	k8s.io/kubectl => k8s.io/kubectl v0.21.1
-	k8s.io/kubelet => k8s.io/kubelet v0.21.1
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.21.1
-	k8s.io/metrics => k8s.io/metrics v0.21.1
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.21.1
 	sigs.k8s.io/cluster-api-provider-aws => github.com/openshift/cluster-api-provider-aws v0.2.1-0.20200520125206-5e266b553d8e // This is coming from machine-api repo
 	sigs.k8s.io/cluster-api-provider-azure => github.com/openshift/cluster-api-provider-azure v0.1.0-alpha.3.0.20200902180535-72169c58a81f
+
```
Comment (aravindhp) [inline go.mod:8]: Nit: not sure if this is needed.

Comment (aravindhp) [reply go.mod:8]: There is also a spelling error in `unecessarily` in the commit message :-)

**Comment by mansikulkarni96** [comment]
Thnaks for working on this @sebsoto 
/lgtm

**Pull request merged**